prototypes/light-mode-test/src/Light/ReadBlocks.hs

```diff
@@ -137,15 +136,13 @@ lightSync follower = do
                 fromBlock <$> (BF.getBlock . Left $ height - stabilityWindow)
     nextBlockFrom prev@(At height _) = do
         b <- hasBeenRolledBack prev
-        case b of
-            False -> fromBlock <$> BF.getLatestBlock
-            True  -> do
-                -- fall back to behind the stability window
-                target <- BF.getBlock . Left $ height - stabilityWindow
-                old <- rollBackward follower $ fromBlock target
-                nextBlockFrom old
-
-    stabilityWindow = 1000 :: Integer
+        (if b then
+             (do target <- BF.getBlock . Left $ height - stabilityWindow
+                 old <- rollBackward follower $ fromBlock target
+                 nextBlockFrom old)
+         else
+             fromBlock <$> BF.getLatestBlock)
```

Comment:
I think the multi-line parenthesis is much harder to read than the original block layout.

Can we keep the old pattern match here?
```suggestion
        case b of
            False -> fromBlock <$> BF.getLatestBlock
            True -> do
                -- fall back to behind the stability window
                target <- BF.getBlock . Left $ height - stabilityWindow
                old <- rollBackward follower $ fromBlock target
                nextBlockFrom old
```

Reply:
sounds reasonable